compiler/damlc/tests/daml-test-files/SetFieldParens.EXPECTED.renamed-daml

```diff
@@ -0,0 +1,3 @@
+SetFieldParens.foo : (Int, Bool) -> (Int, Bool)
+SetFieldParens.foo tup
+= (setField @"_2" not (getField @"_2" tup) tup)
```

Comment:
This is missing a parenthesis around the second argument I think, like this:

```suggestion
= (setField @"_2" (not (getField @"_2" tup)) tup)
```

We may as well try to fix all of these at once I think.